etc/helm/pachyderm/templates/_helpers.tpl

```diff
@@ -134,11 +134,14 @@ true
 
 {{- define "pachyderm.userAccessibleOauthIssuerHost" -}}
 {{- if .Values.oidc.userAccessibleOauthIssuerHost -}}
+  {{- if not (hasPrefix "http" .Values.oidc.userAccessibleOauthIssuerHost) -}}
```

Comment:
IIUC this is a breaking change because users don't currently have to have `http(s)://` but this would require that?
Would it be possible to use the same logic we have for [pachyderm.hostproto](https://github.com/pachyderm/pachyderm/blob/4a47712ebc3d6dafa61e3e86c33a05edb0e45e57/etc/helm/pachyderm/templates/_helpers.tpl#L39-L45) to avoid a breaking change?
Maybe something like this? btw I have no idea if this would work or is even possible.
```
  {{- if not (hasPrefix "https" .Values.oidc.userAccessibleOauthIssuerHost) -}}
    {{- if or .Values.ingress.tls.enabled .Values.proxy.tls.enabled .Values.ingress.uriHttpsProtoOverride -}}
    <add https:// prefix to userAccessibleOauthIssuerHost>
    {{- else if not (hasPrefix "http" .Values.oidc.userAccessibleOauthIssuerHost) -}}
    <add http:// prefix to userAccessibleOauthIssuerHost>
```

Reply:
+1

Reply:
yeah, i don't think we should make a change to the format if `.Values.oidc.userAccessibleOauthIssuerHost`, just to `pachyderm.userAccessibleOauthIssuerHost`
so I think it shouldn't even check for http(s).. just automatically add `pachyderm.hostproto`

Reply:
there should never be a situation where `userAccessibleOauthIssuerHost` has a different proto than `pachyderm.hostproto`